### ISSUE TITLE
acts_as_votable not compatible with Rails 6.1?

### ISSUE THREAD

**Issue by ramonpm** [ryanto/acts_as_votable#212]
I'm trying to upgrade my project to Rails 6.1, but now I'm seeing a lot of these warnings and some of my RSPEC tests are not rolling back the database even with `config.use_transactional_fixtures = true`, looks related to this warning.

```
DEPRECATION WARNING: Using `return`, `break` or `throw` to exit a transaction block is
deprecated without replacement. If the `throw` came from
`Timeout.timeout(duration)`, pass an exception class as a second
argument so it doesn't use `throw` to abort its block. This results
in the transaction being committed, but in the next release of Rails
it will rollback.
```

**Comment by Harshit11071998** [comment]
The gem is also not available at https://rubygems.org

**Comment by ryanto** [comment]
I just released a new version (0.13.0) that works with Rails 6.1.

Let me know if it works for you.
